src/Symfony/Component/HttpKernel/EventListener/DebugHandlersListener.php

```diff
@@ -27,6 +27,8 @@
  * @author Nicolas Grekas <p@tchwork.com>
  *
  * @final
+ *
+ * @internal
```

Comment:
```suggestion
 * @internal since Symfony 5.3
```

Reply:
@derrabus Thanks for the fix. I applied it directly to the PR, I really hope you don't mind. I did not want to bloat the commit history for minor fix like that.